src/plugins/console/common/constants/welcome_panel.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const kibanaApiExample = `
+# retrieve sets of saved objects 
+POST kbn:api/saved_objects/_export
+`;
+
+export const multipleRequestsExample = `
+# POST test_index/_doc/test_doc 200 OK
+{
+ "_index": "test_index",
+ "_id": "test_doc",
+ "_version": 3,
+ "result": "updated",
+ "_shards": {
+   "total": 2,
+   "successful": 1,
+   "failed": 0
+ },
+ "_seq_no": 2,
+ "_primary_term": 1
+}
+# POST notAnEndpoint 405 Method Not Allowed
+{
+ "error": "Incorrect HTTP method for uri [/notAnEndpoint?pretty=true] and method [POST], allowed: [HEAD, GET, PUT, DELETE]",
+ "status": 405
+}
+`;
+
+export const commentsExample = `
+# This request searches all of your indices.
+GET _search
+{
+ // The query parameter indicates query context.
+ "query": {
+   // Matches all documents.
+   /*"match_all": {
+     "boost": 1.2
+   }*/
+   "match_none": {} // Matches no document.
+ }
+}
+`;
+
+export const variablesExample = `
+GET $\{pathVariable}
+{
+ "query": {
+   "match": {
+     "$\{bodyNameVariable}": "$\{bodyValueVariable}"
+   }
+ }
+}
+`;
+
+export const keyboardShortcutsImageUrl =
+  'https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt3e9d63b81228222c/639ba8008f1f170dcb41233d/auto-ident-command.gif';
```

Comment:
gif seems to be stuttering a bit? @gchaps 

https://user-images.githubusercontent.com/1191206/215084259-29fa8677-490f-45d1-a801-c1372f6b8fb1.mov

Reply:
@andreadelrio Can you take a look at the gif?

Reply:
Unfortunately, that gif was made for a POC. I don't have the expertise to enhance it further.

Reply:
So maybe we just remove the entire section on keyboard shortcuts.

Reply:
I agree with removing the keyboard shortcuts section since we have a dedicated help page for that

Reply:
I've removed this section now.